include/utils.h

```diff
@@ -780,6 +801,59 @@ inline void load_aligned_bin(const std::string &bin_file, T *&data, size_t &npts
     }
 }
 
+inline void load_aligned_bin(const std::string &bin_file, std::any &data, size_t &npts, size_t &dim,
+                             size_t &rounded_dim, const std::string &data_type)
```

Comment:
please take a second look at this method

Reply:
removed, its not being used.